wordpress-mobile/WordPress-Android
# Analytics - Add properties to comment reply

Fixes #6207 by adding `blog_id` and `comment_id` properties for the blog and comment that the reply is in reference to.

To test:
Reply to a comment in the app (or quick reply to a comment) and check Tracks Live View for the event `wpandroid_notifications_replied_to`.  You should see your event with the 2 properties in. `blog_id` is top level, instead `comment_id` is at "event level" by design.
